README.md
# simplelightbox
Touch-friendly image lightbox for mobile and desktop with jQuery

### Features
* responsive
* touchfriendly
* swipe gestures for next/previous image
* easy to install, easy to use
* minimalistic
* Only some css is included. You can change the style like you want!
* lots of options
* preloading next and previous image
* Android, iOs and Windows phone support
* CSS3 Transitions with fallback for older browsers
* Works in every modern Browser, even in IE 9+
* Can use jQuery 1.x,2.x and 3.x
* Keyboard support
* Pinch to zoom
* Double-tap to zoom

### Install
```sh
// YARN
yarn add simplelightbox

//Bower
bower install simplelightbox

//NPM
npm install simplelightbox
```

### Usage
Simple include simplelightbox.css and simple-lightbox.js to your page
```javascript
var lightbox = $('.gallery a').simpleLightbox(options);
```

### JavaScript Options
| Property | Default | Type | Description |
| -------- | ------- | ---- | ----------- |
| sourceAttr | href | string | the attribute used for large images |
| overlay | true | bool | show an overlay or not |
| spinner | true | bool | show spinner or not |
| nav | true | bool | show arrow-navigation or not |
| navText | ['&larr;','&rarr;'] | array | text or html for the navigation arrows |
| captions | true | bool | show captions if availabled or not |
| captionSelector | 'img' | string | set the element where the caption is. Set it to "self" for the A-Tag itself |
| captionType | 'attr' | string | how to get the caption. You can choose between attr, data or text |
| captionsData | title | string | get the caption from given attribute |
| captionPosition | 'bottom' | string | the position of the caption. Options are top, bottom or outside (note that outside can be outside the visible viewport!) |
| captionDelay | 0 | int | adds a delay before the caption shows (in ms) |
| captionClass | '' | string | add an additional class to the sl-caption |
| close | true | bool | show the close button or not |
| closeText | '×' | string | text or html for the close button |
| swipeClose | true | bool | swipe up or down to close gallery |
| showCounter | true | bool | show current image index or not |
| fileExt | 'png&#124;jpg&#124;jpeg&#124;gif' | regexp or false | list of fileextensions the plugin works with or false for disable the check |
| animationSpeed | 250 | int | how long takes the slide animation |
| animationSlide | true | bool | weather to slide in new photos or not, disable to fade |
| preloading | true | bool | allows preloading next und previous images |
| enableKeyboard | true | bool | allow keyboard arrow navigation and close with ESC key |
| loop | true | bool | enables looping through images |
| rel | false | mixed | group images by rel attribute of link with same selector.
| docClose | true | bool | closes the lightbox when clicking outside |
| swipeTolerance | 50 | int | how much pixel you have to swipe, until next or previous image |
| className: | 'simple-lightbox' | string | adds a class to the wrapper of the lightbox |
| widthRatio: | 0.8 | float | Ratio of image width to screen width |
| heightRatio: | 0.9 | float | Ratio of image height to screen height |
| scaleImageToRatio: | false | bool | scales the image up to the defined ratio size |
| disableRightClick | false | bool | disable rightclick on image or not |
| disableScroll | true | bool | stop scrolling page if lightbox is opened |
| alertError | true | bool | show an alert, if image was not found. If false error will be ignored |
| alertErrorMessage | 'Image not found, next image will be loaded' | string | the message displayed if image was not found |
| additionalHtml | false | string | Additional HTML showing inside every image. Usefull for watermark etc. If false nothing is added |
| history | true | bool | enable history back closes lightbox instead of reloading the page |
| throttleInterval | 0 | int | time to wait between slides |
| doubleTapZoom | 2 | int | zoom level if double tapping on image |
| maxZoom | 10 | int | maximum zoom level on pinching |
| htmlClass | 'has-lightbox' | string or false | adds class to html element if lightbox is open. If empty or false no class is set |
### Events
| Name | Description |
| ---- | ----------- |
| show.simplelightbox | this event fires before the lightbox opens |
| shown.simplelightbox | this event fires after the lightbox was opened |
| close.simplelightbox | this event fires before the lightbox closes |
| closed.simplelightbox | this event fires after the lightbox was closed |
| change.simplelightbox | this event fires before image changes |
| changed.simplelightbox | this event fires after image was changed |
| next.simplelightbox | this event fires before next image arrives |
| nextDone.simplelightbox | this event fires after next image was arrived |
| prev.simplelightbox | this event fires before previous image arrives |
| prevDone.simplelightbox | this event fires after previous image was arrived |
| nextImageLoaded.simplelightbox | this event fires after next image was loaded (if preload activated) |
| prevImageLoaded.simplelightbox | this event fires after previous image was loaded (if preload activated) |
| error.simplelightbox | this event fires on image load error |

**Example**  
```javascript
$('.gallery a').on('open.simplelightbox', function () {
  // do something…
});

$('.gallery a').on('error.simplelightbox', function (e) {
  console.log(e); // some usefull information
});
```

### Public Methods
| Name | Description |
| ---- | ----------- |
| open | Opens the lightbox with an given jQuery Element |
| close | Closes current openend Lightbox |
| next | Go to next image |
| prev | Go to previous image |
| destroy | Destroys the instance of  the lightbox |
| refresh | Destroys and reinitilized the lightbox, needed for eg. Ajax Calls, or after dom manipulations |

**Example**  
```javascript
var gallery = $('.gallery a').simpleLightbox();

gallery.next(); // Next Image
```

### Multiple Lightboxes on one page
You can have multiple lightboxes on one page, if you give them different selectors. Here is a small example:
```javascript
var lightbox1 = $('.lighbox-1 a').simpleLightbox();
var lightbox2 = $('.lighbox-2 a').simpleLightbox();
```

### Customization

You can customize Simplelightbox by changing the style in simplelightbox.css.

If you are using SASS, you can customize Simplelightbox with the following variables

```SASS
$sl-font-family: Arial, Baskerville, monospace;
$sl-overlay-background: #fff;
$sl-navigation-color: #000;
$sl-caption-color: #fff;
$sl-caption-background: #000;
$sl-overlay-opacity: 0.7;

$sl-counter-fontsize: 1rem;
$sl-caption-fontsize: 1rem;
$sl-close-fontsize: 3rem;

$sl-breakpoint-medium: 35.5em; // 568px, when 1rem == 16px
$sl-breakpoint-large:  50em;   // 800px, when 1rem == 16px

$sl-arrow-fontsize-small:  2rem;
$sl-arrow-fontsize-medium: 3rem;
$sl-arrow-fontsize-large:  3rem;
$sl-img-border-small:  0 none;
$sl-img-border-medium: 0 none;
$sl-img-border-large:  0 none;
$sl-iframe-border-small:  0 none;
$sl-iframe-border-medium: 0 none;
$sl-iframe-border-large:  0 none;

$add-vendor-prefixes: true !default;
```


### Changelog
**1.17.2 - Fixed caption keeps disappeared on double click #139 and added better close symbol #133**  
**1.17.1 - Added webp in fileExt list #135, removed hardcoded a-tag in isValidLink function #134**  
**1.17.0 - Merged pull request #132. Added double click to zoom for desktop browsers - Thanks to coderkan**  
**1.16.3 - Merged pull request #126,#127 - Thanks to jieter**  
**1.16.2 - Added featured #124 - Add a class to html element if lightbox is open**  
**1.16.1 - Fixed pinch to zoom offset error on scrolling #123**  
**1.16.0 - Pinch to Zoom feature for touch devices with new options doubleTapZoom and maxZoom #79**  
**1.15.1 - Merged pull request #113,#114,#115 - Thanks to RaphaelHaettich and celsius-jochen**  
**1.15.0 - Merged pull request #111, fixed #101 and added possibility to close lightbox on load #74**  
**1.14.0 - Merged pull request #107 and #108. Thanks to RaphaelHaettich**  
**1.13.0 - Added featured #92 and merged pull request #98 and #99. Thanks to RaphaelHaettich**  
**1.12.2 - Bugfix for #89**  
**1.12.1 - Bugfix for #88,#87 and remove bind/unbind #84**  
**1.12.0 - New option captionClass #81, bugfix for #82**  
**1.11.1 - Merged pull request #76. Thanks to walterebert, added support for images with parameters and file extension check #59**  
**1.11.0 - New option for src of image. e.g data attribute #70**  
**1.10.7 - Added Bootstrap compatibility #69**  
**1.10.6 - Merged pull requests #65. Thanks to mstaniuk**  
**1.10.5 - Merged pull requests #60 and #61. Thanks to slavanga**  
**1.10.4 - Bugfix von #58**  
**1.10.3 - Merged pull requests #55, #56 and #57. Thanks to karland**  
**1.10.2 - Aligned navigation and close buttons #51, fixed image error bug #52**  
**1.10.1 - Added support for jQuery 3.x #50**  
**1.10.0 - Implemented feature-request #48, history back, some bugfixing and code styling**  
**1.9.0 - Implemented feature-request #16, added rel option for grouping images**  
**1.8.6 - Implemented feature-request #46, added refresh method**  
**1.8.5 - Implemented feature-request #44**  
**1.8.4 - Bugfix for #41 and added option for additional html inside images #40**  
**1.8.3 - Bugfix for #38 and small other fix for loop false option**  
**1.8.2 - Better bugfix for #33, finally fixing multiple lightbox on one page slowness issues!**  
**1.8.1 - Bugfix for #31, #32 and #33**  
**1.8.0 - New API Events (changed open to show) and little fix in function open() brought by Geoffrey Crofte and some other small bugfixes**  
**1.7.2 - Bugfix von #25 and #27**  
**1.7.1 - Bugfix von #22 with new option alertError and merged pull request #23**  
**1.7.0 - Add support for fading between photos, Bugfix for single image navigation, option for caption delay**  
**1.6.0 - Option for caption position. Disable prev or next arrow if loop is false and position is first or last.**  
**1.5.1 - Bugfix for multiple lightboxes on one page**  
**1.5.0 - Added options for disabling rightclick and scrolling, changed default prev- and next-button text**  
**1.4.6 - Option for fileExt can now be false to enable pictures like example.com/pic/200/100**  
**1.4.5 - Bugfix lightbox opening does not work on mobile devices**  
**1.4.4 - Bugfix no drag&drop in FF, changed default close text, only output data if lightbox is opened**  
**1.4.3 - Bugfix z-index for spinner to low, added sass files**  
**1.4.2 - Bugfix for issue #2 - Drop Event does not fire when mouse leaves window**  
**1.4.1 - The whole caption Selector is rewritten. You can now select an element and get its text, use data or attribute**  
**1.4.0 - Caption Attribute can now be what, you want, or data-title. Fixed some small issues**  
**1.3.1 - Bugfix: disable keyboard control if lightbox is closed**  
**1.3.0 - Added current index indicator/counter**  
**1.2.0 - Added option for captions attribute (title or data-title)**  
**1.1.2 - Bugfix for looping images**  
**1.1.1 - Bugfix for loading indicator and removed a log-event**  
**1.1.0 - Added classname for lightbox wrapper and width/height ratio**  
**1.0.0 - Initial Release**

### Author
[Andre Rinas](https://www.andrerinas.de/) - [Github](https://github.com/andreknieriem/)

### Contributors
[nicekiwi](https://github.com/nicekiwi)  
[helloilya](https://github.com/helloilya)  
[bitstarr](https://github.com/bitstarr)  
[Geoffrey Crofte](http://geoffrey.crofte.fr/) - [Github](https://github.com/creativejuiz/)  
[Karl Anders](http://webseiten-anders.de/) - [Github](https://github.com/karland/)  
[Raphael Hättich](https://github.com/RaphaelHaettich)  
[Jochen Sengier](https://www.celcius.be) - [Github](https://github.com/celcius-jochen/)  


html5qrcodereader-master/changelog.txt
02.03.2017:
- updated jQuery to v3.1.1
- added autoplay option to say cheese
- minified included scripts

18.05.2014:
- replaced linkify function in effects_saycheese.js with a better detection

10.05.2014:
- added reload button
- added link detection for index_saycheese.html

cash.php
<?php 
include("config.php");
$profile_data = mysqli_fetch_assoc(mysqli_query($conn, "SELECT * FROM users WHERE id='".$_SESSION['login']."'"));
// print_R($profile_data);
// die;
if($profile_data['user_roles']==5)
{
	$loginidset=$profile_data['parentid'];
}
else
{

	$loginidset=$_SESSION['login'];

}
$query=mysqli_query($conn,"select *  from cash_system where user_id='$loginidset' and is_active='y'");
$pastquery=mysqli_query($conn,"select *  from cash_system where user_id='$loginidset' and is_active='n' order by id desc");
$totalcount=mysqli_num_rows($query);
 $pastcount=mysqli_num_rows($pastquery);


?>
<!DOCTYPE html>
<html lang="en" style="" class="js flexbox flexboxlegacy canvas canvastext webgl no-touch geolocation postmessage websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths">

<head>
    <?php include("includes1/head.php"); ?>
    <link href="js/bootstrap-datetimepicker/css/bootstrap-datetimepicker.min.css" rel="stylesheet" type="text/css" />
	<style>
	.container {
    height: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
}
table, th, td,tr {
  border: 2px solid gray;
}
.add 
{
	background-color: #50d2b7;
	width: 25px;
	height: 25px;
	font-size: 12px;
	border-radius: 100%;
	text-align: center;
	line-height: normal;
	padding: 4px 0 0 0;
	margin: 0;
	display: inline-block;
	vertical-align: top;
}

button {
  background-color:#51D2B7; /* Green */
  border: none;
  color: white;
  padding: 5px 10px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
}

	</style>
</head>

<body class="header-light sidebar-dark sidebar-expand pace-done">

    <div class="pace  pace-inactive">
        <div class="pace-progress" data-progress-text="100%" data-progress="99" style="transform: translate3d(100%, 0px, 0px);">
            <div class="pace-progress-inner"></div>
        </div>
        <div class="pace-activity"></div>
    </div>

    <div id="wrapper" class="wrapper">
        <!-- HEADER & TOP NAVIGATION -->
        <?php include("includes1/navbar.php"); ?>
        <!-- /.navbar -->
        <div class="content-wrapper">
            <!-- SIDEBAR -->
            <?php include("includes1/sidebar.php"); ?>
            <!-- /.site-sidebar -->
            <main class="main-wrapper clearfix" style="min-height: 522px;">
                <div class="row" id="main-content" style="padding-top:25px">
				 <?php if($totalcount==0){ ?>
				     <div style="width:100%;">
					 	<!--h2>
							Please <span class="start_cash" style="color:#51D2B7">click here</span> to Start Cash System
							</h2!-->
					 </div>
					 
				 <?php } if($totalcount>0){
					 $totaldata=mysqli_fetch_assoc($query);
					 $start_utc=$totaldata['login_time'];
					 $cash_id=$totaldata['id'];
					 $date_time=date("Y-m-d H:i", $start_utc);
					 $show_time=date("H:i A d-m-Y", $start_utc);
					    $saleq="select id,quantity,amount from order_list where status in(0,1,2) and created_on >= '$date_time' and merchant_id='$loginidset'";
					
					  $total_sale = mysqli_query($conn,$saleq);
					  // print_R($total_sale);
					  $finaltotal=0;
					  while ($row=mysqli_fetch_assoc($total_sale)){
						   $product_qty = explode(",", $row['quantity']);
							$product_amt = explode(",", $row['amount']);
							$total=0;
							$c=0;
							foreach($product_amt as $p)
							{
								 $total+=($p*$product_qty[$c]);
								$c++;
								// echo "</br>";
							}
							// echo $total;
							// die;
							$finaltotal+=$total;
							
					  }
					  // echo $finaltotal;
					   // $finaltotal=number_format($finaltotal,2);
					
					 ?>
				 <h4 style="margin-left:32%;">Cash System</h4>
				
				 <p>Opening From <?php echo  $show_time;  ?></p>
				   <div class="container">
							<input type='hidden' id='opening_bal' value='<?php echo $totaldata['opening']; ?>'/>
							<input type='hidden' id='sales' value='<?php echo number_format($finaltotal,2); ?>'/>
							<input type='hidden' id='cash_in' value='<?php echo $totaldata['cash_in']; ?>'/>
							<input type='hidden' id='cash_out' value='<?php echo $totaldata['cash_out']; ?>'/>
							<input type='hidden' id='void_tras' value='<?php echo $totaldata['void_tras']; ?>'/>
							
							<table class="table" style="max-width:450px;">
							  <tbody>
							    <tr>
								  <td>Opening  Balance</td>
								  <td><?php echo  number_format($totaldata['opening'],2);?></td>
								</tr>
								<tr>
								  <td>Sales</td>
								  <td><?php echo $finaltotal; ?></td>
								  <td><a href="orderview.php">Detail</a></td>
								</tr>
								<tr>
								  <td>Cash In <i type="cash_in"  class="add fa fa-plus"></i></td>
								  <td><?php echo  number_format($totaldata['cash_in'],2);?></td>
								  <td class="tras_detail" type="cash_in">Detail</td>
								</tr>
								<tr>
								  <td>Cash Out <i  type="cash_out" class="add fa fa-plus"></i></td>
								  <td><?php echo  number_format($totaldata['cash_out'],2);?></td>
								  <td class="tras_detail" type="cash_out">Detail</td>
								</tr>
								<tr>
								  <td>Void Trascation <i type="void_tras"  class="add fa fa-plus"></i></td>
								  <td><?php echo  number_format($totaldata['void_tras'],2);?></td>
								  <td class="tras_detail" type="void_tras">Detail</td>
								</tr>
								<tr>
								  <td>Balance </td>
								  <td><?php $balance=$totaldata['opening']+$finaltotal+$totaldata['cash_in']-($totaldata['cash_out']+$totaldata['void_tras']);
									echo number_format($balance, 2);
									$cash_id=$totaldata['id'];
									 $query="UPDATE cash_system SET balance='$balance',sales='$finaltotal' WHERE `cash_system`.`id`='$cash_id'";
							$update=mysqli_query($conn,$query);
								  ?>
								  <input type='hidden' id='balance' value='<?php echo $balance; ?>'/>
								  </td>
								  <td></td>
								</tr>
							  </tbody>
							  
							  
							</table>
							<!--button onclick="ShiftClose()">Shift Close</button!-->
							<button><a href="orderview.php" style="color:white;">Return</a></button>
					</div>
					   <div style="width:100%">   
				 <?php  } if($pastcount>0){?>
					<h4>Past Cash System Record</h4>
					<div class="col-sm-2" style="margin-top:2%;">
					<a href="print_cash_report.php"  class="btn btn-primary form-control" >Report</a>
					
					</div>
					<?php  if($pastcount>1){?>
					<div class="col-sm-2" style="margin-top:2%;">
					
					<a href="print_cash_report.php?type=past"  class="btn btn-primary form-control" >All Report</a>
					</div>  
					<?php } ?>
					   <table class="table">
						<thead>
						  <tr>
							<th>Opening Balance</th>
							<th>Sales</th>
							<th>Cash in</th>
							<th>Cash out</th>
							<th>Void Trascation</th>
							<th>Balance</th>
							<th>Start time</th>
							<th>End time</th>
						  </tr>
						</thead>
						<tbody>
						<?php   while ($p=mysqli_fetch_assoc($pastquery)){?>
						    <tr>
							<th><?php echo number_format($p['opening'],2); ?></th>
							<th><?php echo number_format($p['sales'],2); ?></th>
							<th><?php echo number_format($p['cash_in'],2); ?></th>
							<th><?php echo number_format($p['cash_out'],2); ?></th>
							<th><?php echo number_format($p['void_tras'],2); ?></th>
							
							<th><?php
							$balancepast=$p['opening']+$p['sales']+$p['cash_in']-($p['cash_out']+$p['void_tras']);
							echo number_format($balancepast,2); ?></th>
							<th><?php echo $start_dt=date("Y-m-d H:i:s", $p['login_time']); ?></th>
							<th><?php echo date("Y-m-d H:i:s", $p['logout_time']); ?></th>
							
						  </tr>
						<?php } ?>
						</tbody>
					  </table>
					<?php } ?>
					</div>
				</div>
			</main>
        </div>
    </div>
	
	<div id="ShiftModel" class="modal fade" role="dialog">
				  			<div class="modal-dialog">

							    <!-- Modal content-->
							    <div class="modal-content">
									   
											<div class="modal-header">
												<button type="button" class="close" id='sectionclose' data-dismiss="modal">&times;</button>
												   Set Opening Balance to Start
											</div>
											 <div class="modal-body" style="padding-bottom:0px;">
											   <div class=" col-md-6 form-group">
													<label>Opening Balance :</label>
													<input type="Number" class="form-control" id='opening_balance' name="opening_balance" value="" placeholder="Opening Balance" required>  
												 <input type="hidden" name="merchant_id" id="merchant_id" value="<?php echo $loginidset;?>">
												 </br>
												 <button type="submit" class="btn btn-primary start_show">Start</button>
												
												</div>
												
											 </div>
										
										
									
					    		</div>

				  			</div>
	</div>
	 <div id="CashDetail" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" style="display: none;">
                                    <div class="modal-dialog" style="max-width:1000px !important;">
									
									<form method="post">
									
									 
                                        <div class="modal-content catelog_plan_body">
										<div class="modal-header">
												<button type="button" class="close" id='sectionclose' data-dismiss="modal">&times;</button>
												  
											</div>
                                           <?php
										   // echo "SELECT * FROM cash_flow WHERE cash_id='".$cash_id."' order by tras_utc desc";
										  
												$cquery = mysqli_query($conn, "SELECT * FROM cash_flow WHERE cash_id='".$cash_id."' order by tras_utc desc");

											$totalc=mysqli_num_rows($cquery);
										   ?>
										      <div class="modal-body">
                                                <div class="row">
												
                        <div class="col-12 table-responsive catelog_body">   
						            <label>Trascation Detail</label>
									<?php if($totalc>0){ ?>
                                    <table id="example24" class="display nowrap table table-hover table-striped table-bordered" cellspacing="0" width="100%">
                                        <thead>
                                         
                                                <tr>
							<th>Amount</th>
							<th>Cash Type</th>
							
							<th>Paid From</th>
							<th>Invoice No</th>
							<th>Cash Description</th>
							<th>Remark</th>
							<th>Trsacation Time</th>
							
						  </tr>
                                                
                                                
                                              
                                           
                                        </thead>
                                       
                                        <tbody> 
										  <?php   while ($cp=mysqli_fetch_assoc($cquery)){?>
						   <tr>
						   <td><?php echo $cp['amount']; ?></td>
						   <td><?php echo $cp['cash_type']; ?></td>
						  
						   <td><?php echo $cp['paid_from']; ?></td>
						    <td><?php echo $cp['invoice_no']; ?></td>
						   <td><?php echo $cp['cash_description']; ?></td>
						   <td><?php echo $cp['remark']; ?></td>
						  <th><?php echo date("Y-m-d H:i:s", $cp['tras_utc']); ?></th>
							
						   </tr>
						<?php } ?>
                                        </tbody>
                                    </table>
									<?php } else { echo "No Category";} ?>
                                </div>

                       
                    
							</div>
							 
														</div>
													
                                        </div>
									</form>
                                    </div>
          </div>
	<div id="CashDetail2" class="modal fade" role="dialog">
				  			<div class="modal-dialog">

							    <!-- Modal content-->
							    <div class="modal-content">
									   
											<div class="modal-header">
												<button type="button" class="close" id='sectionclose' data-dismiss="modal">&times;</button>
												  
											</div>
											 <div class="modal-body" id="report_detail" style="padding-bottom:0px;">
											  <?php 

$cquery = mysqli_query($conn, "SELECT * FROM cash_flow WHERE cash_id='".$cash_id."'");

 $totalc=mysqli_num_rows($query);
?>
<?php if($totalc>0){ ?>
  <table class="table">
		<thead>
						  <tr>
							<th>Amount</th>
							<th>Cash Type</th>
							<th>Invoice No</th>
							<th>Paid From</th>
							<th>Cash Description</th>
							<th>Remark</th>
							
						  </tr>
						</thead>
						<?php  ?>
						<tbody>
						<?php   while ($cp=mysqli_fetch_assoc($cquery)){?>
						   <tr>
						   <td><?php echo $cp['amount']; ?></td>
						   <td><?php echo $cp['cash_type']; ?></td>
						   <td><?php echo $cp['invoice_no']; ?></td>
						   <td><?php echo $cp['paid_from']; ?></td>
						   <td><?php echo $cp['cash_description']; ?></td>
						   <td><?php echo $cp['remark']; ?></td>
						  
						   </tr>
						<?php } ?>
						</tbody>
	</table>
<?php  } else { echo "No data";}?>
	
												
											 </div>
										
										
									
					    		</div>

				  			</div>
	</div>
    
	<div id="AddModel" class="modal fade" role="dialog">
				  			<div class="modal-dialog">

							    <!-- Modal content-->
							    <div class="modal-content">
									   
											<div class="modal-header">
												<button type="button" class="close" id='sectionclose' data-dismiss="modal">&times;</button>
												  <p id="page_title"> Set Opening Balance to </p>
											</div>
											 <div class="modal-body" style="padding-bottom:0px;">
											   <form action="cashadd.php" method="post">
												<div class="form-group">
												  
												  <input type="hidden" class="form-control"  name="cash_id" value="<?php echo $cash_id; ?>">  
												  <input type="hidden" class="form-control"  name="cash_type" id="cash_type">  
												  <input type="hidden" class="form-control"  name="sales" value='<?php echo  $finaltotal;?>'>  
												  <input type="text" class="form-control"  name="amount"  placeholder="Amount" required>  
												</div>
												<div class="form-group" id="invoice_area" style="display:none;">
												  
												  <input type="text" class="form-control"  name="invoice_id" placeholder="Invoice Id">  
												</div>
												<div class="form-group" id="paid_area" style="display:none;">
												  
												  <input type="text" class="form-control"  name="paid_from" placeholder="Paid From">  
												</div>
												<div class="form-group">
												  <label>Description</label>
												 <textarea rows="4" cols="40" name="cash_description"></textarea> 
												</div>
												<div class="form-group">
												   <label>Remark</label>
												  <input type="text" class="form-control"  name="remark"  placeholder="Remark">  
												</div>
												<button type="submit" name="Add" class="btn">Add</button>
											  </form>
																							
												
											 </div>
										
										
									
					    		</div>

				  			</div>
	</div>
    <!-- /.widget-bg -->
  
    <!-- /.content-wrapper -->
    <?php include("includes1/footer.php"); ?>
	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css">

	<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
	<script src="js/bootstrap-datetimepicker/js/bootstrap-datetimepicker.min.js" type="text/javascript"></script>
	<!-- <script src="js/components-date-time-pickers.min.js" type="text/javascript"></script> -->
</body>

</html>
<script>
$(document).ready(function() {
	$('.start_cash').click(function() {
		$('#ShiftModel').modal('show');
	});
	$('.tras_detail').click(function() {
		var type=$(this).attr('type');
		var cash_id='<?php echo $cash_id; ?>';
		var data = {cash_id:cash_id};
		$('#CashDetail').modal('show');
		
	});
	$('.add').click(function() {
		var type=$(this).attr('type');
		$('#cash_type').val(type);
		if(type=="cash_in")
		{
			title="Add Cash In";
			$('#paid_area').show();
			$('#invoice_area').hide();
		}
		if(type=="cash_out")
		{
			title="Add Cash Out";
			$('#paid_area').show();
			$('#invoice_area').hide();
		}
		if(type=="void_tras")
		{
			title="Add Void Trascation";
			$('#invoice_area').show();
			$('#paid_area').hide();
		}
		$('#page_title').html(title);
		$('#AddModel').modal('show');
	});
	$('.start_show').click(function() {
		var opening_balance=$('#opening_balance').val();
		var user_id=$('#merchant_id').val();
		// alert(user_id);
		if(opening_balance)
		{
			var data = {user_id:user_id,opening_balance:opening_balance,method:"startcash"};
			$.ajax({
				  
				  url :'functions.php',
				  type:'POST',
				  dataType : 'json',
				  data:data,
				  success:function(response){
					  var data = JSON.parse(JSON.stringify(response));
					  if(data.status==true)
					  {
						location.reload();  
					  }
					  else
					  {
						  alert('Failed to start cash system');
					  }
					 
					}		  
			  });
		}
		else
		{
			alert('Opening Balance is Required To start');
		}
	});
});
function ShiftClose() {
  var txt;
  var r = confirm("Are You sure want to close Shift!");
  if (r == true) {
	  var user_id=$('#merchant_id').val();
	  var opening=$('#opening').val();
	  var sales=$('#sales').val();
	  // alert(sales);
	  var cash_in=$('#cash_in').val();
	  var cash_out=$('#cash_out').val();
	  var void_tras=$('#void_tras').val();
	  var balance=$('#balance').val();
      var cash_id='<?php echo $cash_id; ?>';
	  var data = {user_id:user_id,cash_id:cash_id,method:"shiftclose",opening:opening,sales:sales,cash_in:cash_in,cash_out:cash_out,void_tras:void_tras,balance:balance};
	  $.ajax({
				  
				  url :'functions.php',
				  type:'POST',
				  dataType : 'json',
				  data:data,
				  success:function(response){
					  var data = JSON.parse(JSON.stringify(response));
					  if(data.status==true)
					  {
						location.reload();  
					  }
					  else
					  {
						  alert('Failed to Close Shift system');
					  }
					 
					}		  
			  });
  } else {
    
  }
 
}
</script>

<style>
.dataTables_wrapper {
    width: 100%;
}
</style>
